src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -149,14 +149,6 @@ int halide_hexagon_remote_set_performance(
 
     HAP_power_request_t request;
 
-    request.type = HAP_power_set_apptype;
```

Comment:
Why remove this? If someone calls set_performance directly, this won't happen... is that a problem?

Reply:
Maybe we should set the compute client class in load_library?